aeron-system-tests/src/test/java/io/aeron/cluster/ClusterTest.java

```diff
@@ -1125,6 +1127,146 @@ void shouldCatchUpTwoFreshNodesAfterRestart()
         cluster.awaitServicesMessageCount(messageCount);
     }
 
+    @Test
+    @InterruptAfter(15)
+    void shouldLogErrorOnBadEgressConfiguration()
+    {
+        cluster = aCluster().withStaticNodes(3).start();
+        systemTestWatcher.cluster(cluster);
+        systemTestWatcher.ignoreErrorsMatching(s ->
+            s.contains("io.aeron.driver.exceptions.InvalidChannelException: " +
+            "java.net.UnknownHostException: unresolved - endpoint=badhost:5555"));
+
+        systemTestWatcher.ignoreErrorsMatching(s ->
+            s.contains("io.aeron.exceptions.RegistrationException: ERROR - " +
+            "java.net.UnknownHostException: unresolved - endpoint=badhost:5555"));
+
+        systemTestWatcher.ignoreErrorsMatching(s ->
+            s.contains("(-9) unknown host"));
+
+
+        final TestNode leader = cluster.awaitLeader();
+
+        final AeronCluster.Context clientContext = cluster.clientCtx()
+            .egressChannel("aeron:udp?endpoint=badhost:5555")
+            .messageTimeoutNs(500_000_000);  // .5s to be safe;
+
+        final String mappings = "badhost,localhost,localhost|node0,localhost,localhost|node1," +
+            "localhost,localhost|node2,localhost,localhost";
+        final RedirectingNameResolver nameResolver = new RedirectingNameResolver(mappings);
+
+        final TimeoutException clusterTimeoutException =
+            assertThrows(TimeoutException.class, () ->
+            {
+                cluster.connectClient(clientContext, nameResolver, true, true);
+            }
+        );
```

Comment:
This should not use an existing `AeronCluster` API provided by the `TestCluster`. Instead create a separate `AeronCluster` instance and if necessary start a separate media driver instance (i.e. `io.aeron.test.driver.TestMediaDriver#launch`).